src/components/files/MilkdownEditor.vue

```diff
@@ -41,7 +53,33 @@ watch(
 onMounted(create);
 onBeforeUnmount(destroy);
 
+const onUpload = async (file: File): Promise<string> => {
+  const parts = props.path.split('/').filter(Boolean);
+  const baseFolder = parts.slice(0, -1).join('/');
+  const path = `${baseFolder}/${file.name}`;
+  await proj?.importFile(path, file.stream());
+  await new Promise((r) => setTimeout(r, 100)); // Otherwise the image won't load
+  await proj?.syncFs({ path: path });
+  return path;
+};
+
+const proxyDomURL = async (url: string): Promise<string> => {
+  const path = decodeURIComponent(url);
```

Comment:
Don't know if this will cause issues.

Reply:
Decided to remove it. Please use full file path instead of URL encoded names. Avoid spaces in file names if possible.